internal/cluster/cluster_suite_test.go

```diff
@@ -11,6 +12,7 @@ import (
 func TestCluster(t *testing.T) {
 	RegisterFailHandler(Fail)
 	common.InitializeDBTest()
+	time.Sleep(10 * time.Second)
```

Comment:
Is this intentional? or just for debugging purposes?

Reply:
Just debugging, removed it now

Reply:
and unheld it as at the time of writing, the hold was intentional but no longer necessary